Comment-Code-Poltergeists/nutshell
# Gc events bugs - Ticket 72

# Description

Ticket 72

Fixes # (issue)

Fixed an issue where the bold text disappeared from the first event item after the event was edited/delete d

## Type of change

Please delete options that are not relevant.



# Testing Instructions
```
git fetch --all
git checkout gc-events bugs
npm start
```

1. Reload the page
2. Make sure that the first event is **bold**
3. Edit/Delete an event from the main container
4. Save the event.
6. Check that the first event item is still bold



# Checklist:
